Asterisk.2013/Asterisk.NET/Manager/Event/QueueMemberPausedEvent.cs

```diff
@@ -1,10 +1,13 @@
+using System;
+
 namespace AsterNET.Manager.Event
 {
 	/// <summary>
 	/// A QueueMemberPausedEvent is triggered when a queue member is paused or unpaused.<br/>
 	/// It is implemented in apps/app_queue.c.<br/>
 	/// Available since Asterisk 1.2
 	/// </summary>
+    [Obsolete("Removed since Asterisk 13", false)]

```

Comment:
This would cause confusion i suspect for anyone who isn't using Asterisk 13 onwards. AsterNET is compatible with all version pre 14.